framework/src/main/java/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -648,31 +824,31 @@ private static int matchingCloseParen(String s, int openPos, char open, char clo
         if (s.length() <= openPos || s.charAt(openPos) != open) {
             return -1;
         }
-
-        int i = openPos + 1;
-        int depth = 1;
-        while (i < s.length()) {
-            char ch = s.charAt(i++);
-            if (ch == '"') {
-                i--;
-                // TODO: inefficient to re-compute this on every iteration, should be static
-                Pattern stringPattern = anchored("(" + STRING_REGEX + ").*");
-                Matcher m = stringPattern.matcher(s.substring(i));
-                if (!m.matches()) {
-                    break;
+        int count = -1;
```

Comment:
Count is not a descriptive name.  Maybe `bracketDepth` would be better.
However, I wonder if this is necessary.  It seems that `openBracket` is a stack.  So why not use `Stack` and `push` and `pop` and `isEmpty` rather than introducing a new integer variable?